bees.json

```diff
@@ -178,10 +172,7 @@
           "image_link": "https://www.inaturalist.org/photos/4965870"
         }
       ],
-      "active_period": [
-        "--04-01/--05-31",
-        "--09-01/--09-30"
-      ],
+      "active_period": "--04-01/--06-31",
```

Comment:
This should be changed. 6-31 is not a date.

Reply:
```suggestion
      "active_period": "--04-01/--06-30",
```

Reply:
Fixed in https://github.com/mn-pollinators/assets/commit/cd27ab7631c413ccb96fc99384314febf4b1680f